DataFormats/TrackSoA/test/TestReadHostTrackSoA.cc

```diff
@@ -0,0 +1,53 @@
+#include "DataFormats/DetId/interface/DetId.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDAnalyzer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/TrackSoA/interface/TracksHost.h"
+
+namespace edmtest {
+
+  class TestReadHostTrackSoA : public edm::global::EDAnalyzer<> {
+  public:
+    TestReadHostTrackSoA(edm::ParameterSet const&);
+    void analyze(edm::StreamID, edm::Event const&, edm::EventSetup const&) const override;
+    void throwWithMessage(const char*) const;
+    static void fillDescriptions(edm::ConfigurationDescriptions&);
+
+    using TracksOnHost = ::reco::TracksHost;
```

Comment:
Why not
```suggestion
    using TracksHost = ::reco::TracksHost;
```
or simply use directly `::reco::TracksHost` ?